integreat_cms/cms/templates/authentication/login_totp.html

```diff
@@ -27,4 +27,16 @@
                autofocus />
         <button class="btn-blue btn">{% translate "Confirm" %}</button>
     </form>
+    {% if can_use_webauthn %}
+        <div class="inline-flex items-center justify-center w-full">
+            <hr class="w-full h-px my-4 bg-gray-200 border-0 dark:bg-gray-300" />
+            <span class="absolute px-3 font-medium text-gray-900 -translate-x-1/2 bg-white left-1/2">{% translate "or" %}</span>
+        </div>
+        <div class="mb-3 mt-4">
+            <a href="{% url 'public:login_totp' %}" class="btn w-full text-center">
```

Comment:
```suggestion
            <a href="{% url 'public:login_webauthn' %}" class="btn w-full text-center">
```
I think here you want to link to the webauthn login, right?